### ISSUE TITLE
media not uploading

### ISSUE THREAD

**Issue by AdrienneRaniszewski** [ArctosDB/arctos#8102]
**Describe the bug**
can't add media. the progress bar completes, as if it's uploaded, but then just sits there for a very long time. doesn't let me proceed

**To Reproduce**
1. go to attach/upload media
2. select or drag a pdf file
3. watch progress bar complete (takes seconds)
4. wait
5. try again
6. wait
7. try a different pdf
8. wait
9. wait
10. wait
11. give up after 5 minutes
12. try on a PC with same result

**Screenshots**
<img width="729" alt="Screenshot 2024-09-11 at 9 52 38 AM" src="https://github.com/user-attachments/assets/fa270d59-7aaf-430e-8699-e35b0870ec8b">

**Data**
[2024-040-trapline-data.zip](https://github.com/user-attachments/files/16966183/2024-040-trapline-data.zip)

**Desktop (please complete the following information):**
 - OS: latest macOS, latest windowsOS
 - Browser: latest Chrome

**Comment by dustymc** [comment]
That's super-huge for that UI. There are no limits imposed, but I'd mostly expect something to time out more often than not when the scale is in MB. This is a job for SCP. MSB has an allocation (https://web.corral.tacc.utexas.edu/arctos/msb/), looks like 

jldunnum
campmlc
tmayfiel

have access. 

I'm not sure what the protocol is, I suppose use https://github.com/ArctosDB/internal/issues/new?assignees=&labels=Digital+Media+Storage+Request&projects=&template=arctos-digital-media-storage-request.md&title=Arctos+Digital+Media+Storage+Request to request changes?? (@mkoo guidance??)

**Comment by AdrienneRaniszewski** [comment]
hmm. okay. I'll scan at a smaller file size. but I don't think any of the scan setting changed recently. I'll double check though.

**Comment by AdrienneRaniszewski** [comment]
I checked my scan setting and it's 300 dpi which is what we've always done so that the files can be printed if we ever need them again. And the file size is comparable to other things that I know loaded just fine.

**Comment by AdrienneRaniszewski** [comment]
@mkoo Do you have any thoughts? Did we run out of space or something? files of this size have never caused problems before.

**Comment by mkoo** [comment]
@AdrienneRaniszewski As dusty said, it's probably a size limitation for the drag/drop interface on Arctos. I can dig around and see if we can figure out what the max size that is allowed (probably 25, 50 MB). Otherwise you can directly add to your allocation if you have a TACC account. Then the single file size limit,  if there's any, will be in the GBs. I have not looked into this but do you have access ([Issue template for requests starts here)](https://github.com/ArctosDB/internal/issues/new?assignees=&labels=Digital+Media+Storage+Request&projects=&template=arctos-digital-media-storage-request.md&title=Arctos+Digital+Media+Storage+Request)

But in any case, I dont necessarily suggest altering your digitization protocols-- have you uploaded successfully a file of smaller size since you filed this issue? (might be valuable info)

**Comment by AdrienneRaniszewski** [comment]
revisiting this finally. 

It still doesn't work and nothing about our protocol has changed, it just stopped being able to upload. We do need to be able to add media through Arctos though, as I usually have students do this. most of our PDFs are about this size. Though I did find a much larger one that uploaded without any issues. https://arctos.database.museum/media/10823820

Also I searched for the PDF I was trying to load (hoping maybe it went through and I just didn't get the confirmation message) When the search didn't find anything it gives me the option to upload through a different interface. I tried that and got the attached error message. 
<img width="1276" alt="Screenshot 2024-10-08 at 1 31 52 PM" src="https://github.com/user-attachments/assets/ad9cd035-4ca4-4a11-b1f4-2d07098e9f8b">

the way described in my first comment, never resulted in an error message. It just didn't do anything after the progress bar completed.
